tests/test_unauthenticated_reddit.py

```diff
@@ -151,6 +151,14 @@ def test_get_random_subreddit(self):
         self.assertTrue(len(subs) > 1)
 
     @betamax()
+    def test_get_random_submission(self):
```

Comment:
Could you do a test similar to get_random_subreddit where it ensures that 2 out of three times it receives a different result?